docs/introduction/tutorials_intro.md

```diff
@@ -1,254 +1,29 @@
-# Arm Mbed tutorials
+# Tutorials and official examples
 
-The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides instructions for specific tasks you may wish to perform. This section contains two formats of documentation: tutorials and examples. Tutorials include step-by-step guidance, and examples are code snippets you can use as a starting point for your application or as a reference on how to use a particular API.
+This page is an index of all tutorials and official examples.
```

Comment:
This page does not list blinky and blinky bare metal but that's probably okay as they are linked in other pages.

Reply:
I did at first link to the quick starts but it just ends up being a third instance of linking to them, and they're already a top-level item, so - as you said - I decided it's a bit much.